src/main/java/com/ldtteam/structurize/placement/StructurePlacer.java

```diff
@@ -323,15 +322,9 @@ public BlockPlacementResult handleBlockPlacement(
             {
                 final List<ItemStack> requiredItems = new ArrayList<>();
 
-                if (!sameBlockInWorld && !this.handler.isCreative())
+                if (!this.handler.isCreative())
                 {
-                    for (final ItemStack stack : placementHandler.getRequiredItems(world, worldPos, localState, tileEntityData, false))
-                    {
-                        if (!stack.isEmpty() && !this.handler.isStackFree(stack))
```

Comment:
no isStackFree check anymore?

Reply:
Good catch.  I had that in the other usage but forgot here.